objax/functional/loss.py

```diff
@@ -71,3 +76,29 @@ def sigmoid_cross_entropy_logits(logits: JaxArray, labels: Union[JaxArray, int])
         (batch, ...) tensor of the cross-entropies for each entry.
     """
     return jn.maximum(logits, 0) - logits * labels + jn.log(1 + jn.exp(-jn.abs(logits)))
+
+
+def mean_squared_error(logits: JaxArray, labels: JaxArray) -> JaxArray:
+    """Computes the mean squared error between logits and labels.
+
+    Args:
+        logits: (batch_size, d0, .. dN) tensor of the predicted values.
+        labels: (batch_size, d0, .. dN) tensor of the ground truth values.
+
+    Returns:
+        (batch_size, d0, .. dN-1) tensor of mean squared error for each entry.
+    """
+    return jn.mean(jn.square(labels - logits), axis=-1)
+
+
+def mean_absolute_error(logits: JaxArray, labels: JaxArray) -> JaxArray:
```

Comment:
Same general comments as for mean squared error.

Reply:
Thanks. Updated both functions as suggested.